app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
the other difference in your pattern is the use of `if` ... `else if` which might get quite long; in these situations I typically prefer `switch`. To each his own I guess.